src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -769,9 +770,52 @@ HRESULT _stdcall TerminalResize(void* terminal, COORD dimensions)
     publicTerminal->_terminal->ClearSelection();
     publicTerminal->_renderer->TriggerRedrawAll();
 
+    // Resize the text buffer without changing the renderer draw size.
     return publicTerminal->_terminal->UserResize(dimensions);
 }
 
+/// <summary>
+/// Resizes the renderer to fit the provided width and height without changing the text
+/// buffer size.
+/// The out parameter "dimensions" is the total amount of characters that would fit with this new size.
+/// </summary>
+HRESULT _stdcall ResizeRendererDrawSpace(_In_ void* terminal, _In_ short width, _In_ short height, _Out_ COORD* dimensions)

```

Comment:
This method basically looks like Refresh with the buffer resize taken out, can we delegate to that method to avoid code duplication?

Reply:
I don't think delegating this to another method is the right move. I think the better option would be to remove this method and add the `AutoFit` parameter to `TriggerResize` so one can choose to also trigger a `UserResize` on resize. That would reduce code duplication and make the code a tad simpler by passing along the `AutoFit` value.